ground/src/main/java/com/google/android/ground/Config.kt

```diff
@@ -44,6 +44,9 @@ object Config {
    */
   const val CLUSTERING_ZOOM_THRESHOLD = 14f
 
+  /** Maximum number of attempts for retrying unsuccessful syncs. */
```

Comment:
It looks like this PR is adding max retry, rather than removing it.. or did I miss something?

Reply:
Ah, sorry about the confusion. Please check to see if the diff view is against HEAD since I accidentally also removed the survey sync retry one in the first pass, then put it back.